packages/react-aria-components/docs/DropZone.mdx

```diff
@@ -256,19 +256,28 @@ To allow the selection of files from the user's device, pass `FileTrigger` as a
 import {FileTrigger, Button} from 'react-aria-components';
 
 function Example() {
-  let [file, setFile] = React.useState(null);
+  let [files, setFiles] = React.useState(null);
 
   return(
-    <DropZone>
+    <DropZone
+      onDrop={(e) => {
+        let files = e.items;
+        let urls = files.map((file) => file.name);
```

Comment:
```suggestion
        let files = e.items.filter((file) => file.kind === 'file') as FileDropItem[];
        let filenames = files.map((file) => file.name);
        setFiles(filenames.join(', '));
```
There seems to be a typescript error because of `file.name`. I think to get rid of that, we should first check to ensure we are getting files and then we can cast the type to be `FileDropItem[]`.  

I also think changing the variable name to `filenames` would reflect what is being captured better than `urls`.

Reply:
Oh forgot to mention you'll need to import `FileDropItem` like so `import type {FileDropItem} from 'react-aria'`